lhotse/cut.py

```diff
@@ -477,7 +502,16 @@ def load_features(self, root_dir: Optional[Pathlike] = None) -> np.ndarray:
             )
         return mixer.mixed_feats
 
-    # TODO: load_audio()
+    def load_audio(self,
+                   recording_set: Optional[RecordingSet] = None,
```

Comment:
Let's remove `recording_set` here too as per previous comment

Reply:
I think this function should be renamed to `load_unmixed_audio`, and there should be one next to it called `load_audio` that applies the offsets and SNRs encoded in `MixTrack`s similarly as in `load_features`